runtime/astar/src/lib.rs

```diff
@@ -1783,6 +1817,38 @@ impl_runtime_apis! {
             use xcm::latest::prelude::*;
             use xcm_builder::MintLocation;
             use astar_primitives::benchmarks::XcmBenchmarkHelper;
+            use pallet_xcm::benchmarking::Pallet as PalletXcmExtrinsicsBenchmark;
+            impl pallet_xcm::benchmarking::Config for Runtime {
+                fn reachable_dest() -> Option<MultiLocation> {
+                    Some(Parent.into())
+                }
+
+                fn teleportable_asset_and_dest() -> Option<(MultiAsset, MultiLocation)> {
+                    // Relay/native token can be teleported between AH and Relay.
```

Comment:
What's AH? AssetHub? 😅

Reply:
This should be `None` imo, we don't support teleports